Automattic/simplenote-ios
# Navigation Bar Blur

### Details:
This PR implements Blur in every single Navigation Bar we've got. How awesome is that?

@etoledom Me again!!! (Thank you!!!!)
Closes #486

### Test: List
1. Perform a fresh install
2. Log into your account

- [x] Verify the Notes List's Navigation Bar has a Blur Background effect

### Test: Editor
1. Log into your account
2. Press over any note to display the editor

- [x] Verify the Notes Editor's NavigationBar has a Blur Background Effect
- [x] Verify that scrolling over a long note reduces the NavigationBar's size (and the blur's height too!)

### Test: Markdown
1. Log into your account
2. Open any random note 
3. Enable Markdown (top right **i** icon)
4. Swipe from the right to the left

- [x] Verify the Markdown Preview UI's NavigationBar has a super cool Blur effect

### Test: Collaborators
1. Log into your account
2. Open any random note
3. Press over the top right icon (i)
4. Press over `Collaborators`

- [x] Verify the Collaborator's NavigationBar has a Blur Effect


### Test: Options
1. Log into your account
2. Press the top left icon to reveal the sidebar
3. Press over `Settings`

- [x] Verify the Settings navigationBar has a Blur Effect

### Release
`RELEASE-NOTES.txt` was updated in 33c32f3 with:

> Every Navigation Bar in Simplenote now has a super cool blur effect!


---

### Snapshot Time!

![Blur](https://user-images.githubusercontent.com/1195260/67724775-e7d35500-f9be-11e9-9de8-cf9a8a90c493.gif)
